ui/app/AppLayouts/Wallet/WalletLayout.qml

```diff
@@ -43,15 +43,66 @@ Item {
             Layout.fillHeight: true
             Layout.fillWidth: true
 
-            Text {
-                id: element1
-                text: qsTr("Send")
-                anchors.left: parent.left
+            TextField {
+                id: txtValue
+                x: 19
+                y: 41
+                placeholderText: qsTr("Enter ETH")
                 anchors.leftMargin: 24
-                anchors.top: parent.top
-                anchors.topMargin: 24
-                font.weight: Font.Bold
-                font.pixelSize: 20
+                anchors.topMargin: 32
+                width: 239
+                height: 40
+            }
+
+            TextField {
+                id: txtFrom
+                x: 340
+                y: 41
+                width: 239
+                height: 40
+                text: assetsModel.getDefaultAccount()
+                placeholderText: qsTr("Send from (account)")
+                anchors.topMargin: 32
+                anchors.leftMargin: 24
+            }
+
+            TextField {
+                id: txtTo
+                x: 340
+                y: 99
+                width: 239
+                height: 40
+                text: assetsModel.getDefaultAccount()
+                placeholderText: qsTr("Send to")
+                anchors.topMargin: 32
+                anchors.leftMargin: 24
+            }
+
+            TextField {
+                id: txtPassword
+                x: 19
+                y: 99
+                width: 239
+                height: 40
+                text: "0x2cd9bf92c5e20b1b410f5ace94d963a96e89156fbe65b70365e8596b37f1f165"
```

Comment:
The password value is the one that is hardcoded in `test.nim` as well. This is just a placeholder for now.